### PR TITLE
ci: update pullapprove

### PR DESCRIPTION
**Pull request by ocombe**
## PR Type
What kind of change does this PR introduce?

```
[x] CI related changes
```

## What is the current behavior?
Tobias is still listed on pullapprove but he's not working on Angular anymore

## What is the new behavior?
Removed Tobias from the list and promoted secondaries to primary when possible, or @chuckjaz when relevant

## Does this PR introduce a breaking change?
```
[x] No
```

## Other information
We need to talk about promoting more people for pullapprove because some packages don't even have a secondary anymore

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -284,7 +279,7 @@ groups:
       files:
         - "packages/benchpress/*"
     users:
-      - tbosch #primary
+      # needs primary
```
Comment (gkalpak) [inline .pullapprove.yml]: I think @alxhub is primary now.

Comment (chuckjaz) [reply .pullapprove.yml]: Corect alxhub is now primary for `benchpress`.

Comment (ocombe) [reply .pullapprove.yml]: ok nice I'll update it

**Comment by vicb** [comment]
On a side note I think that pullapprove functionality is now builtin into GH. Could you take a quick look at pros/cons @ocombe ?

**Comment by ocombe** [comment]
From what I understand from their [very short doc](https://help.github.com/articles/about-codeowners/) it is pretty similar. The main difference is that github will automatically ask a review to the code owners when the PR touches their code, when pullapprove lets you add whoever you want.
This is a big difference for us, because we add multiple owners per branch as backups (usually Igor & Misko) in case the primary/secondary owners are not there.
Also we don't want to ask all of the owners for each PR, only the primary or secondary is fine. That would create a lot of noise for nothing.

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**